modules/planning/common/open_space_info.cc

```diff
@@ -23,5 +23,6 @@
 namespace apollo {
 namespace planning {
 OpenSpaceInfo::OpenSpaceInfo() {}
+// load open space info to open space info debug
```

Comment:
This .cc is so dummy. You can define it as `= default;` in the header file.

Reply:
> This .cc is so dummy. You can define it as `= default;` in the header file.

fixed